### PR TITLE
Release/10.2.0

### PR DESCRIPTION
**Pull request by sidharthv96**
## :bookmark_tabs: Summary

Brief description about the content of your PR.

Resolves #<your issue id here>

## :straight_ruler: Design Decisions

Describe the way your implementation works or what design decisions you made if applicable.

### :clipboard: Tasks

Make sure you

- [ ] :book: have read the [contribution guidelines](https://github.com/mermaid-js/mermaid/blob/develop/CONTRIBUTING.md)
- [ ] :computer: have added unit/e2e tests (if appropriate)
- [ ] :notebook: have added documentation (if appropriate)
- [ ] :bookmark: targeted `develop` branch

### CODE REVIEW

File: packages/mermaid/src/diagrams/class/parser/classDiagram.jison [hunk 1]
```diff
@@ -200,7 +200,7 @@ Function arguments are optional: 'call <callback_name>()' simply executes 'callb
 
 start
     : mermaidDoc
-    | statments
+    | statements
```
Comment (sidharthv96) [inline packages/mermaid/src/diagrams/class/parser/classDiagram.jison:203]: @jgreywolf @knsv this error comes when building jison. 

But I have no clue how this used to work when it was `statments` 🤷‍♂️ 

```
Conflict in grammar: multiple actions possible when lookahead token is $end in state 4
- reduce by rule: statement -> direction
- reduce by rule: start -> direction
Conflict in grammar: multiple actions possible when lookahead token is NEWLINE in state 4
- reduce by rule: statement -> direction
- reduce by rule: start -> direction

States with conflicts:
State 4
  start -> direction . #lookaheads= $end NEWLINE
  statement -> direction . #lookaheads= $end NEWLINE
```

Comment (sidharthv96) [reply packages/mermaid/src/diagrams/class/parser/classDiagram.jison:203]: I have added [a fix](https://github.com/mermaid-js/mermaid/pull/4338), can you verify? @knsv @jgreywolf

**Comment by sidharthv96** [comment]
Released [10.2.0-rc.1](https://www.npmjs.com/package/mermaid/v/10.2.0-rc.1)

**Comment by sidharthv96** [comment]
Published [10.2.0-rc.3](https://www.npmjs.com/package/mermaid/v/10.2.0-rc.3)

**Comment by sidharthv96** [comment]
Don't merge yet, one review pending in https://github.com/mermaid-js/mermaid/pull/4361#discussion_r1181409092

**Comment by mrueg** [comment]
@sidharthv96 sorry to ask here, is there any timeline when 10.2.0 will be released? Looking for a single file distribution that I can include in one of my apps.

**Comment by sidharthv96** [comment]
We're planning to do it this week.  🤞

**Pull request merged**